tail/tail.go

```diff
@@ -57,6 +58,46 @@ type State struct {
 	Offset int64
 }
 
+// GetSampledEntries wraps GetEntries and returns a list of channels that
+// provide sampled entries
+func GetSampledEntries(conf Config, sampleRate uint) ([]chan string, error) {
+	unsampledLinesChans, err := GetEntries(conf)
+	if err != nil {
+		return nil, err
+	}
+	if sampleRate == 1 {
```

Comment:
we're sure `sampleRate` will be `1` here and that we shouldn't be checking `<=`? (what *is* the normal boundary for "if someone tells us '0' for sample rate they really mean '1'"?